config/pmd-test.xml

```diff
@@ -50,6 +50,8 @@
     <exclude name="JUnitUseExpected"/>
     <!-- will be addressed in https://github.com/checkstyle/checkstyle/projects/5 -->
     <exclude name="NonThreadSafeSingleton"/>
+    <!-- Clashes with CommentDefaultAccessModifier. Also, we have the same violation in Sonar. -->
```

Comment:
<!--
Clashes with CommentDefaultAccessModifier.
we do not like package and protected visibility, they are HACK in design and illusion of protection of method from outside. Keeping them public is ok.
-->

Reply:
done